src/error-display/error-display-data.tsx

```diff
@@ -92,9 +92,9 @@ const ImgPaths: Record<string, ImagePathAttributes> = {
         lg: "https://assets.life.gov.sg/react-design-system/img/error/transfer-unsuccessful@3x.png",
     },
     "unsupported-browser": {
-        base: "https://assets.life.gov.sg/react-design-system/img/error/generic-error.png",
-        md: "https://assets.life.gov.sg/react-design-system/img/error/generic-error@2x.png",
-        lg: "https://assets.life.gov.sg/react-design-system/img/error/generic-error@3x.png",
+        base: "https://assets.life.gov.sg/react-design-system/img/error/unsupported-browser.png",
```

Comment:
unsupported-browser.png is still in the old illustration style. I recall it was intended to point to the new generic-error.png